packages/rgbpp/src/rgbpp/summary/asset-summarizer.ts

```diff
@@ -0,0 +1,98 @@
+import { Cell } from '@ckb-lumos/base';
+import { Utxo, encodeUtxoId } from '@rgbpp-sdk/btc';
+import { leToU128, encodeCellId } from '@rgbpp-sdk/ckb';
+
+export interface AssetSummary {
+  amount: bigint;
+  utxos: number;
+  cells: number;
+}
+
+export interface AssetGroupSummary {
+  utxoId: string;
+  cellIds: string[];
+  assets: Record<string, AssetSummary>;
+}
+
+export interface TransactionGroupSummary {
+  utxos: number;
+  cells: number;
+  utxoIds: string[];
+  cellIds: string[];
+  assets: Record<string, AssetSummary>;
+}
+
+export class AssetSummarizer {
+  groups: AssetGroupSummary[] = [];
+
+  constructor() {}
+
+  addGroup(utxo: Utxo, cells: Cell[]): AssetGroupSummary {
+    const utxoId = encodeUtxoId(utxo.txid, utxo.vout);
+    const assets: Record<string, Omit<AssetSummary, 'xudtTypeArgs'>> = {};
```

Comment:
The `AssetSummary` interface does NOT contain a `xudtTypeArgs` property.
Maybe the `Omit` type is unnecessary in this context?

Reply:
Fixed at 1440fb8